src/AbstractArray.php

```diff
@@ -368,6 +368,18 @@ public function createClone()
     }
 
     /**
+     * Return slice of an array except given keys.
+     * 
+     * @param array $keys List of keys to exclude
+     * 
+     * @return AbstractArray The created array
+     */
+    public function except(array $keys)
+    {
+        return new static(array_diff_key($this->elements, array_fill_keys($keys, 1)));
```

Comment:
What about to use `array_flip($keys)` instead of `array_fill_keys($keys, 1)` here? I don't have strong reasons to do it, but I think it's more readable and I'd like `except()` and `only()` methods were consistent with each other.

What do you think?